bin/generateModel.py

```diff
@@ -75,13 +55,13 @@ def main():
     group_optional = optparse.OptionGroup(parser, "Optional")
 
     group_required.add_option('-d', '--data-set',
-                              help="Data set to use for model generation.")
-    group_required.add_option('-c', '--content-descriptor',
-                              help="Feature descriptor type for model and "
-                                   "feature generation.")
-    group_required.add_option('-i', '--indexer',
-                              help="(Optional) Indexer type for model "
-                                   "generation.")
+        help="Data set to use for model generation.")
+    group_required.add_option('-c', '--content-detect-and-describe',
+        help="Feature descriptor type for model and feature generation.")
+    group_required.add_option('-q', '--quantization',
```

Comment:
Make this a more generic description. A quantizer, or the quantization step, just refers to what amounts to a dimensionality reduction action. In our case, bringing a 2D NxM matrix into an M-length vector. Codebooks are an implementation specific concept.